dist/highlight.css

```diff
@@ -0,0 +1,3 @@
+.knotx-devtool-highlight {
```

Comment:
this file couldn't be in /dist directory. Please move it to another location or use inline style for this functionality.